API.md

```diff
@@ -1394,6 +1394,49 @@ If an HTTP status code of 200 is returned, the body of the response will contain
 
 If there was an error, expect an HTTP status code in either the 4XX or 5XX range.
 
+### modify
+
+Modifies a single property of a resource.
+
+#### Request
+
+```bash
+curl http://localhost:<port>/irods-http-api/<version>/resources \
+    -H 'Authorization: Bearer <token>' \
+    --data-urlencode 'op=modify' \
+    --data-urlencode 'name=<string>' \ # Name of the resource.
+    --data-urlencode 'property=<string>' \ # Name of the property to modify.
+    --data-urlencode 'value=<string>' # The new value of the property.
+```
+
+The following properties are supported:
+- name
+- type
+- host
+- vault_path
+- context
+- status
+- free_space
+- comments
+- information
+
+For the `status` property, `value` must be set to `up` or `down`. Attempting to pass a value other the ones described will result in an HTTP error.
```

Comment:
This does not feel correct.   iRODS does not limit this value to 'up' or 'down'.

Reply:
Right, but the administration library only allows for `up` or `down` (https://github.com/irods/irods/blob/5e0e0c8dd5534b337cbced8a4c6ebbb39d363e51/lib/administration/resource/include/irods/resource_administration.hpp#L77-L83).